docs/cookbook/localprocessing.rst

```diff
@@ -4,3 +4,78 @@ Client-side (local) processing
 
 .. warning::
     This is a new experimental feature and API, subject to change.
+
+Background
+-----
+
+The client-side processing functionality allows to test and use openEO with its processes locally, i.e. without any connection to an openEO back-end.
+It relies on the projects `openeo-pg-parser-networkx <https://github.com/Open-EO/openeo-pg-parser-networkx>`__, which provides an openEO process graph parsing tool, and `openeo-processes-dask <https://github.com/Open-EO/openeo-processes-dask>`__, which provides an xArray and Dask implementation of most openEO processes. 
+
+Installation
+-----
+
+**Please note:** this feature requires ``Python>=3.9`` and has been tested
```

Comment:
instead of a custom `**Please note:**`, use RST "admonitions":

```
.. important::
    this feature requires ...
```

see https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions   for  more details and what types are available (caution, error, hint, tip, ...)